tuned/plugins/plugin_video.py

```diff
@@ -124,3 +199,27 @@ def _get_radeon_powersave(self, device, ignore_missing = False):
 			return "dpm-" + self._cmd.read_file(sys_files["dpm_state"]).strip()
 		else:
 			return None
+
+	@command_set("panel_power_savings", per_device=True)
+	def _set_panel_power_savings(self, value, device, sim, remove):
```

Comment:
With `sim=True`, the function should do precisely what it would do with `sim=False` except for actually overwriting the file. It can still print warnings and more importantly, it should return the same value. Currently, it always returns `None` if `sim` is `True`.

Reply:
Fixed